packages/web-api/src/setup-ioc-container.ts

```diff
@@ -11,5 +11,8 @@ export function setupIoContainer(): inversify.Container {
     registerLoggerToContainer(container);
     registerAzureServicesToContainer(container);
 
+    container.unbind(AzureServicesIocTypes.CredentialType);
+    container.bind(AzureServicesIocTypes.CredentialType).toConstantValue(CredentialType.AppService);
+
```

Comment:
Can we add optional param to registerAzureServicesToContainer() func with option of CredentialType to have correct bind handled inside registerAzureServicesToContainer()